src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -0,0 +1,227 @@
+
+* :**Destination Name::** Custom name to identify the destination
+* :**Jira account endpoint::** the URL of the destination
+* :**User-name:** this will be the email address of the user making the connection
+* :**API Token:** generated from your Atlassian account
+
+<Callout variant="important">
+  New Relic currently supports company-managed projects.
+</Callout>
+
+Before saving the destination, we recommend you test the connection via the **test connection** button.
+
+## Two-way sync [#jira-two-way]
+
+To enable two-way sync between Jira and New Relic One, enable the `two-way integration` toggle.
+
+When enabled, a Jira webhook would be created in your Jira account at a later stage, for the selected project (see ‘customize a message template’ below). The webhook would contain access details to Newrelic (URL and Newrelic API-KEY).
+
+By default, any update to a Jira issue created by New Relic One will sync back to New Relic One. After enabling two-way integration, when Jira issue changes to closed the corresponding workflow issue will change to Closed.
+
+## Configure the message template [#configure-jira-message-template]
+
+Upon successful connection, Jira fields are fetched from your account and automatically mapped to your Jira instance. 
+
+To help you get started, recommended fields and values are automatically presented. You can add or remove fields and change the values to customize the notification to your needs.
+
+![A screenshot of the Jira-specific custom variables menu.](./images/jira-custom-variables.png "The Jira-specific custom variables menu.")
+
+<figcaption>The variables menu changes depending on the destination integration you're using.</figcaption>
+
+## Send a test notification [#send-test-notification]
+
+Users can see how the JIRA issue will appear by clicking a test notification with default field values. If successful, a Jira ticket is created and a link to that ticket appears. 
+  
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="servicenow"
+    title="ServiceNow"
+  >
+  Integrate New Relic with ServiceNow and automatically create and update incidents.
+
+## Roles [#roles]
+
+For this integration to work, your ServiceNow user details need read permissions for the tables: `sys_dictionary`, `sys_choice`, `sys_user`, and `change_request` (optional).
+
+You need a ServiceNow admin role enable two-way integration between ServiceNow and New Relic One.
+
+## Set up a destination [#set-up-service-destination]
+
+To create a ServiceNow destination, enter the following information:
+
+* **Name:** custom name to identify the destination
+* **Domain:** the URL of the destination
+* **User-name:**  the name of the user
+* **Password:**  the user name’s password 
+
+## Two-way Sync [#two-way-servicenow]
+
+You can configure a two-way integration with ServiceNow incidents so that when state updates for the incident (resolved or closed), it triggers an update in the corresponding New Relic issue state. 
```

Comment:
```suggestion
You can configure a two-way integration with ServiceNow incidents so that state updates for the incident (resolved or closed) trigger an update in the corresponding New Relic issue state. 
```